test/common/alerts_exist.go

```diff
@@ -861,12 +879,22 @@ func TestIntegreatlyAlertsExist(t TestingTB, ctx *TestingContext) {
 	}
 }
 
-func getExpectedAWSRules(installType string, installationName string) []alertsTestRule {
-	if rhmiv1alpha1.IsRHOAMMultitenant(rhmiv1alpha1.InstallationType(installType)) {
-		return commonExpectedAWSRules(installationName)
-	} else {
-		return append(commonExpectedAWSRules(installationName), managedApiAwsExpectedRules(installationName)...)
+func getExpectedCloudPlatformRules(ctx *TestingContext, installType, installationName, platformType string) ([]alertsTestRule, error) {
+	useClusterStorage, err := isClusterStorage(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("error getting isClusterStorage: %w", err)
+	}
+	expectedRules := getExpectedRules(installType, installationName)
+	if !useClusterStorage {
+		expectedRules = append(expectedRules, commonExpectedCloudPlatformRules(installationName)...)
+		if rhmiv1alpha1.IsRHOAMSingletenant(rhmiv1alpha1.InstallationType(installType)) {
```

Comment:
@adam-cattermole Do we need to handle when RHOAM is multitenant here? Should this instead be using `IsRHOAM()`?
```suggestion
		if rhmiv1alpha1.IsRHOAM(rhmiv1alpha1.InstallationType(installType)) {
```

Reply:
So I think this was an oversight from the E2E PR 5220 - the `managedApiCommonExpectedRules` weren't being added at all. These used to be called `managedApiAwsExpectedRules` and were only added if the cluster was single tenant so I've replicated the behaviour here. This is where they are added on master -> https://github.com/integr8ly/integreatly-operator/blob/master/test/common/alerts_exist.go#L864-L870